data/component-design-tokens/filter-chip-design-tokens.json

```
@@ -0,0 +1,127 @@
+{
+  "filter-chip-border-radius": {
+    "value": "{borderRadius.2xl}",
+    "type": "borderRadius"
+  },
+  "filter-chip-border": {
+    "value": {
+      "color": "{color.greyscale.300}",
+      "width": "{borderWidth.s}",
+      "style": "solid"
+    },
+    "type": "border"
+  },
+  "filter-chip-border-color-hover": {
+    "value": "{color.greyscale.500}",
+    "type": "color"
+  },
+  "filter-chip-text-color": {
+    "value": "{color.text.default}",
+    "type": "color"
+  },
+  "filter-chip-color-bg": {
+    "value": "{color.greyscale.white}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-hover": {
+    "value": "{color.greyscale.100}",
+    "type": "color"
+  },
+  "filter-chip-text-color-error": {
+    "value": "{input.color.border.error}",
+    "type": "color"
+  },
+  "filter-chip-text-color-error-hover": {
+    "value": "{input.color.text.error}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-error": {
+    "value": "{input.color.background.error}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-error-hover": {
+    "value": "{input.color.background.error-hover}",
+    "type": "color"
+  },
+  "filter-chip-border-color-error": {
+    "value": "{input.color.border.error}",
+    "type": "color"
+  },
+  "filter-chip-border-color-error-hover": {
+    "value": "{input.color.border.error-hover}",
+    "type": "color"
+  },
+  "filter-chip-icon-color-error": {
+    "value": "{input.color.border.error}",
```

```suggestion
    "value": "{input.color.text.error}",
```

In error state, the leading icon should use the text error color (darker, more visible) rather than the border error color.